sawtooth_sdk/consensus/engine.py

```diff
@@ -62,3 +62,11 @@ def name(self):
         Return:
             str
         '''
+
+    @abc.abstractproperty
+    def additional_protocols(self):
```

Comment:
This might benefit from having a default implementation.

Reply:
This is a good idea; updated